src/Titles.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from "react";
+
+class Titles extends Component {
+    render() {
+        const titles = ["Make", "Model", "Model year", "Color", "Dealer e-mail"];
```

Comment:
It would make this component much more generic, if this information came from `props`. After all it should be up to the parent component to decide what titles it wishes to render in the table header.